src/gpuarray_collectives_cuda_nccl.c

```diff
@@ -53,8 +54,25 @@ static void comm_free(gpucomm* comm) {
 
 static const char* comm_error(gpucontext* c) {
   cuda_context* ctx = (cuda_context*) c;
-  // find a way to concatenate a constact "(nccl) " infront
-  return ncclGetErrorString(ctx->nccl_err);
```

Comment:
If you can get an error string from nccl, then we can forget about the "(nccl) " in front.  cublas doesn't offer a cublasGetErrorString() so that's why it went this way.